frontend/src/app/models/income-source.command.ts

```diff
@@ -0,0 +1,5 @@
+export interface IncomeSourceCommand {
```

Comment:
idk if that's a good idea, but maybe `IncomeSourceCommand` could extend `IncomeSourceModel`?
TS types could help here to catch things like refactoring the model and forgetting about the command. I'm more thinking out loud, then really suggesting it, just to have your input on this :)